SimCalorimetry/EcalSimProducers/src/EcalDigiProducer_Ph2.cc

```diff
@@ -198,12 +198,10 @@ void EcalDigiProducer_Ph2::accumulateCaloHits(HitsHandle const& ebHandle, int bu
 
 void EcalDigiProducer_Ph2::accumulate(edm::Event const& e, edm::EventSetup const& eventSetup) {
   // Step A: Get Inputs
-  edm::Handle<std::vector<PCaloHit>> ebHandle;
 
   m_EBShape.setEventSetup(eventSetup);
   m_APDShape.setEventSetup(eventSetup);
-  edm::InputTag ebTag(m_hitsProducerTag, "EcalHitsEB");
-  e.getByLabel(ebTag, ebHandle);
+  const edm::Handle<std::vector<PCaloHit>>& ebHandle = e.getHandle(m_HitsEBToken);
 
   accumulateCaloHits(ebHandle, 0);
 }
```

Comment:
There is another getByLabel below in line 216. https://github.com/cms-sw/cmssw/pull/37773/files#diff-00d232f260111e56efbec17ef741ffab3c9b78dd638786b123215036552b0c50R216

Reply:
Please see the earlier answer.